frontend/app/src/components/staking/kraken/KrakenStakingEvents.vue

```diff
@@ -105,11 +116,7 @@ const useHeaders = () => {
       align: 'end'
     },
     {
-      text: i18n
-        .t('kraken_staking_events.column.value', {
-          symbol: unref(currencySymbol)
-        })
-        .toString(),
+      text: '',
```

Comment:
I was using the same approach in the past and this results in an empty header when the table switches to the mobile layout, so it would be better to keep it